cosmic_ray/testing/test_runner.py

```diff
@@ -66,4 +68,4 @@ def __call__(self):
                                   test_result[1])
         except Exception as e:
```

Comment:
Since we're no longer using `e`, we don't need to capture it in the `except` clause.